src/v2/view-builder/utils/FactorUtil.js

```diff
@@ -45,14 +41,13 @@ const factorData = {
   },
 
   'security_key': {
-    label: loc('oie.webauthn', 'login'),
     description: loc('oie.webauthn.description', 'login'),
     iconClassName: 'mfa-webauthn',
   },
 
   'app': {
     label: loc('oie.app', 'login'),
```

Comment:
@magizh-okta line #49 - I have not removed it like other authenticators, so the label is hard-coded to come from the props file. Want me to change? ( I will change the key but I may have to remove it here if we're taking the same route as other authenticators )